### ISSUE TITLE
A few suggestions/fixes for widgets

### ISSUE THREAD

**Issue by icpryde** [SwagOtaku/plugin.video.kaito#4]
Alright, so since this is mainly using Seren I figured I would bring this up as the widgets part could use a bit of work let me explain.

On Seren, I have the widget for "next up" showing the next episode of what I am watching and it shows the latest episode, when I click it, it just plays that episode as seen here:

![Screen Shot 2020-10-18 at 4 41 05 PM](https://user-images.githubusercontent.com/29389746/96385279-d996bf80-1160-11eb-910e-9af49884e340.png)

Now when I set a Kaito widget for AniList with "Currently Watching" and set it to lanscape view to see the episode thumbnail...this is what happens:

![Screen Shot 2020-10-18 at 4 23 39 PM](https://user-images.githubusercontent.com/29389746/96385299-f8955180-1160-11eb-8631-44f97a3511c1.png)

YOu will also notice it doesn't show the thumbnail for the episode and under the thumbnail it doesn't mention the episode you are on either but it does mention it above when highlighted.

Now when I change the widget to "poster" view this works well because it shows the poster like this:

![Screen Shot 2020-10-18 at 4 25 05 PM](https://user-images.githubusercontent.com/29389746/96385326-2a0e1d00-1161-11eb-872b-38ff2143c500.png)

Now this isn't to bad, at least we can see the poster of the show and watch it by click it as when you click it, it takes you to the episode list. then you have to click the episode you want to watch. The Seren version will just start loading the correct episode without those additional steps.

So what I was hoping, if its possible to be able to put the widget in "landscape" and it finds the latest episode watched and displays that thumbnail on the widget....the thumbnails are available as you can see here:

![Screen Shot 2020-10-18 at 4 31 04 PM](https://user-images.githubusercontent.com/29389746/96385377-748f9980-1161-11eb-9dca-e011f711e981.png)

I was just hoping it could be closer to the "next up" widget like seren to show that episode thumbnail and bonus.....if possible make it just play from the widget instead of it just taking you to the "episode list" and lastly show the season and episode number as well on the widget.

**Comment by SwagOtaku** [comment]
@icpryde Added 'Next Up' menu item to watchlist menu items in version 0.0.5. Let me know how it is.

**Comment by icpryde** [comment]
@SwagOtaku So Next Up seems to work, when i now click the episode it will play that specific episode now BUT one little issue....it doesn't load the thumbnail artwork at all this is what I see now for the shows.

![Screen Shot 2020-10-25 at 6 33 46 PM](https://user-images.githubusercontent.com/29389746/97120878-c780c800-16f0-11eb-98bb-c99154fdf426.png)

Again when I go to and click on the actual show and then it lists the episodes I can see screenshots from there:

![Screen Shot 2020-10-25 at 6 35 27 PM](https://user-images.githubusercontent.com/29389746/97120897-e7b08700-16f0-11eb-8c73-df93ac7bb39a.png)

**Comment by icpryde** [comment]
@SwagOtaku another thing, when I use next up and I finish the episode the up next pop up shows up when I click it to play the next episode it doesn’t work and just goes back to the main screen of kodi.

**Comment by icpryde** [comment]
@SwagOtaku I can confirm version 0.0.6 fixes this and now the thumbnails appear properly as widgets for Next Up. 

![Screen Shot 2020-11-27 at 9 20 56 PM](https://user-images.githubusercontent.com/29389746/100492014-76e8fa00-30f6-11eb-946c-cefb7ced2334.png)

The episode plays as intended as well!

One issue still remains, once the episode is complete the Up Next feature to display the next episode displays something completely incorrect. I was watching Digimon on episode 11 and then at the end of the episode instead of suggesting episode 12 it showed a movie instead.

![Screen Shot 2020-11-27 at 9 19 00 PM](https://user-images.githubusercontent.com/29389746/100492002-59b42b80-30f6-11eb-86a8-0cca7c1f252f.png)

Even when I clicked on it, then it went back to the main screen where my widgets are and then nothing happened after that.

That seems to be the only bug left for this feature.

Amazing work!

**Comment by icpryde** [comment]
So I attempted another episode using Next Up and it played perfectly again and then at the end of the episode when Next Up popped up to request the next episode it showed the correct episode.

![Screen Shot 2020-11-28 at 6 21 21 PM](https://user-images.githubusercontent.com/29389746/100528068-86c01700-31a6-11eb-9a0e-0223691362ed.png)

Though when I went back to another show using Next Up it and tried Digimon the episode worked but first it doesn't show the artwork or the episode information when you pause, it simply shows this.

![Screen Shot 2020-11-28 at 6 23 12 PM](https://user-images.githubusercontent.com/29389746/100528086-c981ef00-31a6-11eb-9580-6cd03a718a01.png)

Also at the end of the episode for Digimon it didn't request the correct show or episode for the next episode.

![Screen Shot 2020-11-28 at 6 24 00 PM](https://user-images.githubusercontent.com/29389746/100528096-e61e2700-31a6-11eb-85ba-8b2e06e652f4.png)

The last thing is when i attempt to play an episode through any other method like clicking current, then the episode i want to play and i pause i can see the poster and information as seen here.

![Screen Shot 2020-11-28 at 6 25 04 PM](https://user-images.githubusercontent.com/29389746/100528115-0bab3080-31a7-11eb-9bed-7b97b95b9ec5.png)

It only doesn't show the information when you do it through Next Up.

So just 2 things for Next Up to summarize:

1) When playing the next episode Up Next doesn't always show the proper episode.
2) The poster and episode information doesn't display.

@SwagOtaku
